references/classification/train.py

```diff
@@ -124,6 +133,7 @@ def parse_args():
                                      formatter_class=argparse.ArgumentDefaultsHelpFormatter)
 
     parser.add_argument('data_path', type=str, help='path to dataset folder')
+    parser.add_argument('use_OpenFire', type=bool, default=False, help='use our open source OpenFire dataset')
```

Comment:
always use lower case in arguments name :)
and here you would need to do `--use_OpenFire True` for this to work

I'd suggest
```
parser.add_argument('use-openfire', dest="use_openfire", action="store_true", help='use our open source OpenFire dataset')
```

so that we only have to do `--use-openfire`